engine/apps/oss_installation/utils.py

```diff
@@ -80,5 +84,7 @@ def cloud_user_identity_status(connector, identity):
         else:
             status = CloudSyncStatus.SYNCED_PHONE_NOT_VERIFIED
 
-        link = urljoin(connector.cloud_url, f"a/grafana-oncall-app/?page=users&p=1&id={identity.cloud_id}")
```

Comment:
this previously had a bug in it, `?page=<page>` URL format is no longer supported